python instagram.py -u <username> -p <passlist>

```diff
@@ -0,0 +1 @@
+
```

Comment:
File "instagram.py", line 176
    def prune_database(prune: float) -> None: what i do?

Reply:
you can simple run `python3.9 instagram.py`

Reply:
Can anyone hack me someone password please I will pay I need a hacker

Reply:
ya mr.holiday1991

Reply:
> Can anyone hack me someone password please I will pay I need a hacker

Did you ever find a hacker for IG? I need one

Reply:
Actually hacking social media is something that is easy, depending on the victim whether stupid enough to be tricked or otherwise. For the rest,  here's my email zivalp84@gmail.com

Reply:
if anyone needs hacker for IG and is willing to pay message me on discord svist#7400